src/current/v23.1/molt-verify.md

```diff
@@ -0,0 +1,110 @@
+
+- **Table Verification:** Check that the structure of tables between the source database and the target database are the same.
+- **Column Definition Verification:** Check that the column names, data types, constraints, nullability, and other attributes between the source database and the target database are the same. 
+- **Row Value Verification:** Check that the actual data in the tables are the same between the source database and the target database.
+
+## Supported databases
+
+The following databases are currently supported:
+
+- PostgreSQL
+- MySQL
+
+## Use the MOLT Verify tool
+
+1. Get the JDBC connection strings for the source and target databases you want to compare.
+2. Make sure the SQL user running the MOLT verify tool has access to the necessary tables.
+3. Install and build the tool: 
+
+    Run the following commands on your local machine:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    git clone https://github.com/cockroachdb/molt.git
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd molt
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    go build .
+    ~~~
+    
+    To cross-compile, use:
+    
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    GOOS=linux GOARCH=amd64 go build -v .
+    ~~~ 
+    
+    Run the following command to verify the installation:
+	  
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ./molt verify --help
+    ~~~
+
+4. Run MOLT Verify: 
+
+    The `molt verify` command takes two or more JDBC connection strings as arguments. You can append a name for easier readability using `<name>===` in front of the JDBC string. The first argument is considered as the "source of truth". 
+    
+    Examples:
+
+    The following example compares a PostgreSQL database with a CockroachDB database (simplified naming for both instances):
+    
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ./molt verify \
+      'pg_truth===postgres://<username>:<password>@url:<port>/<database>' \
+      'crdb_compare===postgresql://<username>:<password>@<host>:<port>/<database>?sslmode=verify-full'
+    ~~~
+
+    The following example compares a MySQL database with a CockroachDB database (simplified naming for both instances):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ./molt verify \
+      'mysql===jdbc:mysql://root@tcp(<host>:<port>)/<database>' \         
+      'postgresql://<username>:<password>@<host>:<port>/<database>?sslmode=verify-full'
+    ~~~
+
+    Optionally, you can use [supported flags](#supported-flags) to customize the verification results.
+
+5. Review the verification results:
+
+    Running the MOLT Verify tool will show if there are any missing rows or extraneous tables in the target database. If any data is missing, you can add the missing data to the target database and run `./molt verify` again.
```

Comment:
is there anything we can link to for "add the missing data"?

Reply:
ah, good catch. Linking to the INSERT statement doc.